pkgs/by-name/na/navidrome/package.nix

```
@@ -10,10 +10,12 @@
   stdenv,
   ffmpeg-headless,
   taglib,
+  xorg,
   zlib,
   nixosTests,
   nix-update-script,
   ffmpegSupport ? true,
+  plugins ? [ ],
 }:
 
 buildGo124Module rec {
```

```suggestion
let
  buildGoModule = buildGo124Module;
in
buildGoModule rec {
```
we might as well update to 125 or the default go version